vis-cmds.c

```diff
@@ -470,7 +470,7 @@ static bool print_keybinding(const char *key, void *value, void *data) {
 	const char *desc = binding->alias;
 	if (!desc && binding->action)
 		desc = binding->action->help;
-	return text_appendf(txt, "  %-15s\t%s\n", key, desc ? desc : "");
+	return text_appendf(txt, "  %-15s\t%s\n", isspace(key[0]) ? "<Space>" : key, desc ? desc : "");
```

Comment:
Ammended on my branch, thanks.

Marc André Tanner <notifications@github.com> writes:

>> @@ -470,7 +470,7 @@ static bool print_keybinding(const char *key, void *value, void *data) {
>>  	const char *desc = binding->alias;
>>  	if (!desc && binding->action)
>>  		desc = binding->action->help;
>> -	return text_appendf(txt, "  %-15s\t%s\n", key, desc ? desc : "");
>> +	return text_appendf(txt, "  %-15s\t%s\n", isspace(key[0]) ? "<Space>" : key, desc ? desc : "");
>
> Thanks, this should be `key[0] == ' ' ? ...`
>
> I can amend it while pulling, if you want.
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/martanne/vis/pull/228/files/aa858656ef85d805f7a92101400e1d6763376212..104c808a0c509a7c8708d630750ade778f3ed87d#r58903908